tests/functional/helpers.js

```diff
@@ -34,3 +34,12 @@ export const repeatingAnswerChangeLink = (answerIndex) => {
 export const listItemIds = () => {
   return $$("[data-list-item-id]").map((listItem) => listItem.getAttribute("data-list-item-id"));
 };
+
+export const click = async (selector) => {
+  // This was introduced due to a css property on ons-btn when the button is right on the very edge of the screen, webdriverio,
+  // sees that the button is accessible, so does not scroll, but clicks down on the very top of the button which moves down
+  // and just below the mouse, so that when the mouse click is released, it's no longer over the button, and the click silently fails.
+  // This means that when the test comes to do assertions on the following page, they fail, as we never navigated to that page
+  await $(selector).scrollIntoView({ block: "center", inline: "center" });
```

Comment:
👍 
might be worth mentioning in the comment that the css property is on `ons-btn:active` so readers understand thats why the button only shifts out of view as soon as the mouse click begins

Reply:
I've pasted that whole property here and did some editing, hopefully the text makes sense now.